### PR TITLE
btc -> ltc re:dpow

### PR DESCRIPTION
**Pull request by gcharang**
re: https://github.com/KomodoPlatform/developer-docs/issues/334

Please review @smk762 @tonymorony

### CODE REVIEW

**Review by smk762 — APPROVED** [review]
LGTM.

File: docs/basic-docs/start-here/core-technology-discussions/delayed-proof-of-work.md [hunk 1]
```diff
@@ -664,21 +671,22 @@ We invite the reader to consider that as each Smart Chain can support thousands
...
 
-The total yearly cost for the Komodo notary nodes to notarize the KMD main chain into the currently chosen PoW chain, Bitcoin, is approximately ~180 BTC/year (a value of ~\$1.5M USD at the time of the writing of this paper). Funding for the notary nodes to perform this service was raised during the Komodo ICO, and current BTC holdings give us many years to come before we will be required to implement any business models to replenish our BTC funds.
+<!-- The total yearly cost for the Komodo notary nodes to notarize the KMD main chain into the currently chosen PoW chain, Litecoin, is approximately ~180 LTC/year (a value of ~\$1.5M USD at the time of the writing of this paper). Funding for the notary nodes to perform this service was raised during the Komodo ICO, and current LTC holdings give us many years to come before we will be required to implement any business models to replenish our LTC funds.
```
Comment (smk762) [inline docs/basic-docs/start-here/core-technology-discussions/delayed-proof-of-work.md:686]: 180 LTC/year is still around about right by my calculations

Comment (gcharang) [reply docs/basic-docs/start-here/core-technology-discussions/delayed-proof-of-work.md:686]: > 180 LTC/year is still around

thanks! beyond this, I also wasn't sure how to rewrite this paragraph to account for the fact that LTC notas are not as costly  as BTC notas.

especially, this part:
```
 Funding for the notary nodes to perform this service was raised during the Komodo ICO, and current BTC holdings give us many years to come before we will be required to implement any business models to replenish our BTC funds.
 ```
 
 should I comment out just this part?

Comment (smk762) [reply docs/basic-docs/start-here/core-technology-discussions/delayed-proof-of-work.md:686]: Yeah I think comment it out.

**Pull request merged**